pkg/proxy/util/utils.go

```diff
@@ -499,3 +501,24 @@ func RevertPorts(replacementPortsMap, originalPortsMap map[utilnet.LocalPort]uti
 func CountBytesLines(b []byte) int {
 	return bytes.Count(b, []byte{'\n'})
 }
+
+// SlicesEquiv Tests whether two slices are equivalent.  This sorts both slices in-place.
+func SlicesEquiv(lhs, rhs []string) bool {
```

Comment:
Since this is moving to a new package, I think we should fix the API.  Sorting in-place is horrible - we should make the caller do that.

Also, we don't need reflect for this - we can just write the loop.

Reply:
OK, I did that on the other PR! (https://github.com/kubernetes/utils/pull/202). Is this the expected now?

If so, I'll then work on the other parts pointed by @apelisse about the Contains and Remove strings from the slice.

Thanks!